projects/core/directives/dropdown/dropdown.component.ts

```diff
@@ -68,6 +68,8 @@ export class TuiDropdownComponent {
         position$.pipe(takeUntil(destroy$)).subscribe(([top, left]) => {
             this.update(top, left);
         });
+
+        this.updateWidth();
```

Comment:
change to `platformBrowserDynamic().bootstrapModule(AppBrowserModule, { ngZoneRunCoalescing: true});` in main.browser.ts please

Reply:
wait for it https://github.com/Tinkoff/taiga-ui/pull/3530